python/tvm/relay/op/transform.py

```diff
@@ -242,3 +242,34 @@ def where(condition, x, y):
     Note that the shape of condition, x, and y needs to be the same.
     """
     return _make.where(condition, x, y)
+
+
+def split(data, indices_or_sections, axis=0, equal_split=False):
+    """Split input tensor along axis by sections or indices.
+
+    With equal_split being true it splits input into indices_or_sections[0]
+    sections and axis dimension should be a multiple of sections.
+
+    Other wise indices_or_sections is a tuple indicating indices
+    where the input is split on given axis.
+
+    Parameters
+    ----------
+    data : relay.Expr
+        The source array.
+
+    indices_or_sections : tuple of int
+        The indices or number of sections to split.
+
+    axis : int, optional
+        The axis over which to split.
+
+    equal_split : boolean
+        To split into indices_or_section[0] sections.
+
+    Returns
+    -------
+    ret : relay.Expr
+        The computed result.
+    """
+    return _make.split(data, indices_or_sections, axis, equal_split)
```

Comment:
make use of https://github.com/dmlc/tvm/blob/master/python/tvm/relay/expr.py#L216